fall18/p2/main.py

```diff
@@ -0,0 +1,61 @@
+import project
+import math
+
+# Insert getMaximumLand(stateName1, stateName2, stateName3) function here
+# Step1: Retrieve area values for all 3 states using getArea function and assign it to variables.
+# Step2: Use max() function to calculate the max value and store it as a variable
+# Step3: Return the max value that you calculated.
+
+
+# Insert getMinimumPopulationDensity(stateName1, stateName2, stateName3, year) function here
+# Step1: Retrieve area and population values for all 3 states using getArea and the getPopulation function and assign it to variables.
+# Step2: Use min() function to calculate the max value and store it as a variable
+# Step3: Return the min value that you calculated.
+
+
+# Insert predictPopulation(stateName, growthRate, yearA, yearB): function here
+# Step 1: Using the statename retrieve the population for yearA
+# Step 2: Calculate time between yearA and yearB
+# Step 3: Use the math.exp() function in the given formula to calculate final population
+# Step 4: Return the predicted population
+
+
+# Insert calcGrowthRate(stateName, yearA, yearB) function here
+# Step 1: Using the statename retrieve the population for yearA
+# Step 1: Using the statename retrieve the population for yearB
+# Step 2: Calculate time between yearA and yearB
+# Step 3: Use the math.log() function in the given formula to calculate growth rate
+# Step 4: Return the final population
+
+
+if __name__ == "__main__":
+    # Example state names. Please play around by changing the state names

```

Comment:
it would be nice if they had to write a couple things in main that gets tested (with check_output, as in P1).  These would just call our functions, and would give them some points even before they create their own functions

Reply:
I am not sure if I have enough to update because we need to release at Wed but this requires to change test, main and markdown file. Let me deal with other problem first.